### PR TITLE
fix(daemon): export logs, delete workspaces, reenable distributed tests

### PR DESCRIPTION
**Pull request by deepankarm**

### CODE REVIEW

**Review by hanxiao — CHANGES_REQUESTED** [review]
please note that `fix(daemon): miscellaneous` is unacceptable, it does not show any useful information. Our auto Release Note system relies on meaningful commit message, otherwise it is hard for everybody to track what is going on each PR and release.

**Comment by deepankarm** [comment]
> please note that `fix(daemon): miscellaneous` is unacceptable, it does not show any useful information. Our auto Release Note system relies on meaningful commit message, otherwise it is hard for everybody to track what is going on each PR and release.

Makes sense. 
Rebase gone wrong. Creating a new PR would be easier than cherry-picking.

File: jina/peapods/runtimes/jinad/__init__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import asyncio
 from typing import Optional
 
-from .client import PeaDaemonClient
+from .client import PodDaemonClient
```
Comment (hanxiao) [inline jina/peapods/runtimes/jinad/__init__.py:5]: if there is no semantic change on `Pea/PodDaemonClient`, I believe this should stick to `PeadDaemonClient`

Comment (deepankarm) [reply jina/peapods/runtimes/jinad/__init__.py:5]: `PodDaemonClient` calls `/pods` which invokes a `Pod`. Any reason we were using `PeaDaemonClient` here?

**Review by JoanFM — APPROVED** [review]

File: jina/peapods/runtimes/jinad/__init__.py (hunk 1 above)
Comment (hanxiao) [inline jina/peapods/runtimes/jinad/__init__.py:5]:
```suggestion
from .client import PeaDaemonClient
```

File: jina/peapods/runtimes/jinad/__init__.py [hunk 2]
```diff
@@ -17,7 +17,7 @@ def __init__(self, args: 'argparse.Namespace'):
         self.timeout_ctrl = args.timeout_ctrl
         self.host = args.host
         self.port_expose = args.port_expose
-        self.api = PeaDaemonClient(host=self.host, port=self.port_expose, logger=self.logger)
+        self.api = PodDaemonClient(host=self.host, port=self.port_expose, logger=self.logger)
```
Comment (hanxiao) [inline jina/peapods/runtimes/jinad/__init__.py:20]:
```suggestion
        self.api = PeaDaemonClient(host=self.host, port=self.port_expose, logger=self.logger)
```

File: jina/peapods/runtimes/jinad/__init__.py (hunk 1 above)
Comment (hanxiao) [inline jina/peapods/runtimes/jinad/__init__.py:5]:
```suggestion
from .client import PeaDaemonClient
```

**Pull request merged**